spatialmath/point.go

```diff
@@ -162,3 +162,11 @@ func pointVsBoxDistance(b *box, pt r3.Vector) float64 {
 	}
 	return -b.penetrationDepth(pt)
 }
+
+// TODO: add descriptions
+func (p *point) ToPointCloud(options map[string]interface{}) ([]r3.Vector, error) {
+	myVec := r3.Vector{p.pose.Point().X, p.pose.Point().Y, p.pose.Point().Z}
```

Comment:
`return []r3.Vector{p.pose.Point()}` should work here.  Also I'm noticing I really shouldn't have made Point store a Pose since theres no point in storing orientation data 🙃

Reply:
changed.

Reply:
should I edit the struct of Point to remove Pose?

Reply:
This would actually be a good fix too thanks